testing/run_sim_tests.py

```diff
@@ -0,0 +1,258 @@
+"""
+Run conveniently tests against simulator. Tests are run module after module. If any tests fail,
+it will try to re-run those failed test with fresh simulator. Has to be run from firmware/testing directory.
+Do not forget to comment/uncomment line in pytest.ini.
+
+. ENV/bin/activate
+python run_sim_tests.py --help
+python run_sim_tests.py --veryslow                             # run ONLY very slow tests
+python run_sim_tests.py --onetime                              # run ONLY onetime tests (each will get its own simulator)
+python run_sim_tests.py --onetime --veryslow                   # run both onetime and very slow
+python run_sim_tests.py -m test_nfc.py                         # run only nfc tests
+python run_sim_tests.py -m test_nfc.py -m test_hsm.py          # run nfc and hsm tests
+python run_sim_tests.py -m all                                 # run all tests but not onetime and not very slow (cca 40 minutes) - most useful
+python run_sim_tests.py -m all --onetime --veryslow            # run all tests (cca 235 minutes)
+
+
+Onetime/veryslow tests are completely separated form the rest of the test suite.
+When using -m/--module do not expect the --onetime/--veryslow to apply. If --onetime/--veryslow
+is specified, these test will run at the end or alone.
+
+python run_sim_tests.py --collect onetime                      # just print all onetime tests to stdout
+python run_sim_tests.py --collect veryslow                     # just print all veryslow tests to stdout
+python run_sim_tests.py --collect manual                       # just print all manual tests to stdout
+
+Make sure to run manual test if you want to state that your changes passed all the tests.
+"""
+
+import os
+import time
+import glob
+import json
+import pytest
+import atexit
+import signal
+import argparse
+import subprocess
+import contextlib
```

Comment:
- merge these imports, add Copyright to top of file
- would prefer `click` to `argparse`, but not worth changing?